internal/server/team_work_in_progress_test.go

```diff
@@ -83,7 +84,7 @@ func TestGetTeamWorkInProgress(t *testing.T) {
 	vars.Today = time.Time{}
 
 	assert.Equal(teamWorkInProgressVars{
-		Cases:    client.casesByTeam.data,
+		Cases:    client.casesByTeam.data.Cases,
```

Comment:
Probably worth adding `Stats` and `Pagination` data to this too to make sure it's all passed through to the template